### PR TITLE
ACM-10019 - 1. add managedHub property type to cache

### PR DESCRIPTION
**Pull request by SherinV**
### Related Issue

https://issues.redhat.com/browse/ACM-10019

### Description of changes
- Added managedHub propType

### CODE REVIEW

File: pkg/resolver/searchHelper.go [hunk 1]
```diff
@@ -92,6 +92,8 @@ func getWhereClauseExpression(prop, operator string, values []string, dataType s
 	// check if the property is cluster
 	if prop == "cluster" {
 		lhsExp = goqu.C(prop)
+	} else if prop == "managedHub" {
+		return exps
```
Comment (SherinV) [inline pkg/resolver/searchHelper.go:96]: Will be removed from this PR later.

**Review by jlpadilla — APPROVED** [review]
/lgtm
/hold

**Review by jlpadilla — APPROVED** [review]
/lgtm

**Pull request merged**